examples/tests/stack_size_test01/Makefile

```diff
@@ -6,7 +6,9 @@ TOCK_USERLAND_BASE_DIR = ../../..
 # Which files to compile.
 C_SRCS := $(wildcard *.c)
 
-STACK_SIZE := 900
+STACK_SIZE       := 2048
```

Comment:
The point of this test is to try setting a non-default stack size, but this changes it to set to what is already the default stack size.